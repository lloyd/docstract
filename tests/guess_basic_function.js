/**
 * documenation of fairly vanilla functions should be simple.  let's see!
 */

/**
 * Transmorgifies the input
 */
function transmorgify(foo, bar, baz) {
    return baz + foo + bar;
}
