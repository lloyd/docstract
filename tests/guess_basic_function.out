{
  "desc": "documenation of fairly vanilla functions should be simple.  let's see!", 
  "filename": "tests/guess_basic_function.js", 
  "functions": [
    {
      "desc": "Transmorgifies the input", 
      "name": "transmorgify"
    }
  ], 
  "module": "guess_basic_function"
}
